Server/src/main/kotlin/net/horizonsend/ion/server/listeners/bukkit/PlayerJoinListener.kt

```diff
@@ -1,14 +1,37 @@
 package net.horizonsend.ion.server.listeners.bukkit
 
+import net.kyori.adventure.text.format.TextColor
+import net.kyori.adventure.text.minimessage.MiniMessage
+import org.bukkit.GameMode
+import org.bukkit.Material
 import org.bukkit.event.EventHandler
 import org.bukkit.event.EventPriority
 import org.bukkit.event.Listener
 import org.bukkit.event.player.PlayerJoinEvent
+import org.bukkit.inventory.ItemStack
 
 class PlayerJoinListener : Listener {
 	@EventHandler(priority = EventPriority.NORMAL)
 	@Suppress("Unused")
 	fun onPlayerJoinEvent(event: PlayerJoinEvent) {
 		event.joinMessage(null)
+
+		if(!event.player.hasPlayedBefore() && event.player.gameMode == GameMode.SURVIVAL){
+			val armor: Array<ItemStack?> = arrayOf()
+			armor[0] = ItemStack(Material.CHAINMAIL_BOOTS, 1)
+			armor[1] = ItemStack(Material.CHAINMAIL_LEGGINGS, 1)
+			armor[2] = ItemStack(Material.CHAINMAIL_CHESTPLATE, 1)
+			armor[3] = ItemStack(Material.CHAINMAIL_HELMET, 1)
```

Comment:
Why not just put these directly in the `arrayOf()`?

Reply:
Also are you sure you need to specify it as nullable?

Reply:
Idk I copied it off a stack overflow answer

Reply:
> Idk I copied it off a stack overflow answer

Said every programmer ever.

Reply:
LOL

Reply:
Ok I fixed it, ill commit it now